source/libs/page-detect.ts

```diff
@@ -247,49 +242,47 @@ export const isRepo = (): boolean => /^[^/]+\/[^/]+/.test(getCleanPathname()) &&
 	'https://github.com/sindresorhus/refined-github/issues/146',
 	'https://github.com/sindresorhus/notifications/',
-	'https://github.com/sindresorhus/refined-github/pull/148'
+	'https://github.com/sindresorhus/refined-github/pull/148',
+	'https://github.com/sindresorhus/refined-github/milestones/new', // Gotcha for isRepoTaxonomyDiscussionList
+	'https://github.com/sindresorhus/refined-github/issues/new/choose', // Gotcha for isRepoIssueList
+	'https://github.com/sindresorhus/refined-github/issues/templates/edit' // Gotcha for isRepoIssueList
 ];
 export const _isRepoSkipNegatives = true;
 
+export const isRepoTaxonomyDiscussionList = (): boolean => /^labels\/.+|^milestones\/\d+/.test(getRepoPath()!);
+export const _isRepoTaxonomyDiscussionList = [
+	'https://github.com/sindresorhus/refined-github/labels/Priority%3A%20critical',
+	'https://github.com/sindresorhus/refined-github/milestones/1'
+];
+
 export const isRepoDiscussionList = (): boolean =>
 	isRepoPRList() ||
 	isRepoIssueList() ||
-	/^(labels|milestones)\/.+/.test(getRepoPath()!);
-export const _isRepoDiscussionList = [
-	'http://github.com/sindresorhus/ava/issues',
-	'https://github.com/sindresorhus/refined-github/pulls',
-	'https://github.com/sindresorhus/refined-github/pulls/',
-	'https://github.com/sindresorhus/refined-github/pulls/fregante',
-	'https://github.com/sindresorhus/refined-github/issues/fregante',
-	'https://github.com/sindresorhus/refined-github/labels/Priority%3A%20critical',
-	'https://github.com/sindresorhus/refined-github/milestones/1.0',
-	'https://github.com/sindresorhus/refined-github/issues?q=is%3Aclosed+sort%3Aupdated-desc',
-	'https://github.com/sindresorhus/refined-github/pulls?q=is%3Aopen+is%3Apr',
-	'https://github.com/sindresorhus/refined-github/pulls?q=is%3Apr+is%3Aclosed',
-	'https://github.com/sindresorhus/refined-github/issues'
-];
+	isRepoTaxonomyDiscussionList();
+export const _isRepoDiscussionList = skip;
 
-export const isRepoPRList = (): boolean => (getRepoPath() ?? '').startsWith('pulls');
+export const isRepoPRList = (): boolean => String(getRepoPath()).startsWith('pulls');
 export const _isRepoPRList = [
 	'https://github.com/sindresorhus/refined-github/pulls',
 	'https://github.com/sindresorhus/refined-github/pulls/',
-	'https://github.com/sindresorhus/refined-github/pulls/fregante',
 	'https://github.com/sindresorhus/refined-github/pulls?q=is%3Aopen+is%3Apr',
 	'https://github.com/sindresorhus/refined-github/pulls?q=is%3Apr+is%3Aclosed'
 ];
 
-export const isRepoIssueList = (): boolean => {
-	const parts = (getRepoPath() ?? '').split('/');
-	return parts[0] === 'issues' && parts[1] !== 'new' && !/\d/.test(parts[1]); // `issues/fregante` is a list but `issues/1` isn't
-};
-
+// `issues/fregante` is a list but `issues/1`, `issues/new`, `issues/new/choose`, `issues/templates/edit` aren’t
+export const isRepoIssueList = (): boolean =>
+	String(getRepoPath()).startsWith('issues') &&
+	!/^issues\/(\d+|new|templates)($|\/)/.test(getRepoPath()!);
```

Comment:
This line specifically should fix https://github.com/sindresorhus/refined-github/issues/2907#issuecomment-607861547

Reply:
It fixed new but not edit

Reply:
What's edit? Always post a full link with bug reports

Reply:
Do you mean `/sindresorhus/refined-github/issues/templates/edit`?

What do you mean by not fixed? `isRepoIssueList` doesn't match that page, so any feature that's supposed to run on `isRepoIssueList` won't run on that page.

This is tested (via _negative_ tests)

https://github.com/sindresorhus/refined-github/blob/641dbcbcccc0a0a04ed9ec1660a593ed23ff6749/source/libs/page-detect.ts#L252

Reply:
milestone edit